hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -97,5 +106,10 @@ public long get(RuntimeMXBean runtimeMXBean) {
                     }
                 }
         );
+        registeredMetrics.add("runtime.uptime");
+    }
+
+    public static Set<String> getRegisteredMetricNames() {
```

Comment:
If you really want to be lazy; just walk over all probes and filter out the probes you don't want based on their name. If it doesn't start with e.g. 'os' then you ignore it. The probes you like you store in a hashmap.